.github/workflows/e2e.yml

```diff
@@ -1,34 +1,24 @@
-name: CI
+name: E2E
 
 on:
   push:
     branches: [main]
+    paths:
+      - 'packages/**'
+      - 'examples/**'
+      - '.github/workflows/e2e.yml'
   pull_request:
     types: [opened, synchronize]
+    paths:
+      - 'packages/**'
+      - 'examples/**'
+      - '.github/workflows/e2e.yml'
```

Comment:
Nice idea!
Do we need to add `'e2e/**'` too?

Reply:
yes